overviews/core/_posts/2012-09-20-futures.md

```diff
@@ -11,11 +11,11 @@ languages: [ja]
 
 ## Introduction
 
-Futures provide a nice way to reason about performing many operations
-in parallel-- in an efficient and non-blocking way. The idea
-is simple, a `Future` is a sort of a placeholder object that you can
-create for a result that does not yet exist. Generally, the result of
-the `Future` is computed concurrently and can be later collected.
+Futures provide a way to reason about performing many operations
+in parallel-- in an efficient and non-blocking way. 
+A [Future](http://www.scala-lang.org/api/current/index.html#scala.concurrent.Future) 
+is a sort of a placeholder object that you can create for a result that does not yet exist. 
+Generally, the result of the `Future` is computed concurrently and can be later collected.
```

Comment:
Perhaps: `Generally, the value of the `Future` is supplied concurrently and can subsequently be used.`